meta-balena-common/recipes-containers/balena/balena_git.bb

```diff
@@ -150,8 +151,10 @@ do_install() {
 	mkdir -p ${D}/usr/lib/balena
 	install -m 0755 ${WORKDIR}/balena-healthcheck ${D}/usr/lib/balena/balena-healthcheck
 
+  install -d ${D}${sysconfdir}/systemd/system/balena.service.d
+  install -c -m 0644 ${WORKDIR}/balena.conf.storagemigration ${D}${sysconfdir}/systemd/system/balena.service.d/storagemigration.conf
```

Comment:
nit, some messy whitespace here

Reply:
@klutchell I think this comment should be on #2218

Reply:
what do you think about adding an [editorconfig](https://editorconfig.org/) file into the repo? I have added one for the tests here: https://github.com/balena-os/meta-balena/blob/master/tests/suites/.editorconfig